pkg/kmodule/compression.go

```diff
@@ -1,6 +1,7 @@
 // Copyright 2024 the u-root Authors. All rights reserved
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
+//go:build !tinygo
```

Comment:
Can you add a newline before this, so it isn't part of the license declaration?

Reply:
done